browser/main/modals/PreferencesModal/UiTab.js

```diff
@@ -207,6 +215,43 @@ class UiTab extends React.Component {
     }, 2000)()
   }
 
+  /**
+   * Gets the total number of minutes and returns a string in the HH:MM format
+   * @param {Number} time total number of minutes
+   */
+  formatTime(time) {
+    let hour = Math.floor(time / 60)
+    let minute = time % 60
+
+    if (hour < 10) {
+      hour = '0' + hour
+    }
+
+    if (minute < 10) {
+      minute = '0' + minute
+    }
+
+    return `${hour}:${minute}`
+  }
+
+  handleSlider(e, number) {
+    const sliderID = 'rs-range-line-' + number
+    const bulletID = 'rs-bullet-' + number
+
+    const rangeSlider = document.getElementById(sliderID)
+    const rangeBullet = document.getElementById(bulletID)
+
+    if (rangeSlider && rangeBullet) {
+      const bulletPosition = rangeSlider.value / rangeSlider.max
+      rangeBullet.style.left =
+        bulletPosition * 574 + 6 * (1 - bulletPosition) + 'px'
```

Comment:
Can you explain to me where the value `574` come from?

Reply:
We thought that we needed to set the slider position, but I tried removing it and it is working as expected, so I've removed the handleSlider function and we can just use the handleUIChange